.github/workflows/build-and-test-external.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and Test External
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name != 'develop' && github.ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  id-token: write
+  contents: read
+  pull-requests: write
+
+jobs:
+  determine-affected:
+    name: "Turbo Affected"
+    if: ${{contains(needs.determine-affected.outputs.paths, 'ledger-live-desktop') && github.event.pull_request.head.repo.full_name != github.repository }}
+    uses: ./.github/workflows/turbo-affected-reusable.yml
+    with:
+      head_branch: ${{ github.event.pull_request.head.ref || github.event.merge_group.head_ref }}
+      base_branch: ${{ github.event.pull_request.base.ref || github.event.merge_group.base_ref }}
+
+  # External from Fork branches
+  build-desktop-external:
+    name: "[External] Build Desktop"
+    needs: determine-affected
+    if: ${{contains(needs.determine-affected.outputs.paths, 'ledger-live-desktop') && github.event.pull_request.head.repo.full_name != github.repository }}
+    uses: ./.github/workflows/build-desktop-external-reusable.yml
+
+  test-desktop-external:
+    name: "[External] Test Desktop"
+    needs: determine-affected
+    if: ${{contains(needs.determine-affected.outputs.paths, 'ledger-live-desktop') && github.event.pull_request.head.repo.full_name != github.repository }}
+    uses: ./.github/workflows/test-desktop-external-reusable.yml
+
+  build-mobile-external:
+    name: "[External] Build Mobile"
+    needs: determine-affected
+    if: ${{contains(needs.determine-affected.outputs.paths, 'ledger-live-mobile') && github.event.pull_request.head.repo.full_name != github.repository}}
+    uses: ./.github/workflows/build-mobile-external-reusable.yml
+
+  # Final Check required
+  ok:
+    name: "OK"
+    needs:
+      - build-desktop-external
+      - test-desktop-external
+      - build-mobile-external
+    runs-on: ubuntu-22.04
+    if: ${{ always() && github.event.pull_request.head.repo.full_name != github.repository }}
+    steps:
+      - name: Compute info
+        id: info
+        if: always()
+        run: |
+          results=$(echo '${{ toJson(needs) }}' | jq -r '[.[] | .result]')
+          if echo $results | jq -e 'map(select(. != "success" and . != "skipped")) | length > 0'; then
+            echo "failure=true" >> $GITHUB_OUTPUT
+          else
+            echo "success=true" >> $GITHUB_OUTPUT
+          fi
+      - name: Check result
```

Comment:
[SHOULD] Pretty sure you can skip this step and just echo / exit in the previous step. WDYT ?